.c8rc

```diff
@@ -0,0 +1,20 @@
+{
+  "all": true,
+  "src": ["client/src", "server/src"],
+  "exclude": [
+    "**/node_modules/**",
+    "**/*.test.*",
+    "client/dist",
+    "client/src/connection/rest/api",
+    "server/dist",
+    "tools",
+    "server/src/python"
+  ],
+  "reporter": ["html", "text"],
+  "excludeAfterRemap": true,
+  "checkCoverage": true,
+  "lines": 52,
+  "statements": 52,
+  "branches": 71,
+  "functions": 43
```

Comment:
What do you think about adding minimum levels now _just_ to make sure things don't get worse, with the eventual goal being having code coverage at 80%. 

And, do we care about these individual numbers, or do we really only care about lines?